Dockerfile

```
@@ -25,15 +25,22 @@ ENV PORT=5000
 ENV PUID=1000
 ENV PGID=1000
 
-# Install su-exec (for privilege dropping), shadow (for usermod/groupmod), and
-# Python + Apprise for local CLI notifications.
-RUN apk add --no-cache su-exec shadow python3 py3-pip && \
+# Install su-exec (for privilege dropping), shadow (for usermod/groupmod),
+# libarchive-tools/bsdtar (RAR archive extraction via libarchive, which supports both
+# legacy and RAR5 formats including multi-volume sets — Alpine no longer ships a
+# standalone `unrar` package), and Python + Apprise for local CLI notifications.
+RUN apk add --no-cache libarchive-tools py3-pip python3 shadow su-exec && \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)